include/UIChar.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef UICHAR_HPP
+# define UICHAR_HPP
+
+# include <string>
+
+class UIChar {
+
+	public:
+		UIChar(std::string const &);
+		UIChar(UIChar const &) = delete;
+		UIChar(UIChar &&) = delete;
+		UIChar &operator=(UIChar const &) = delete;
+		UIChar &operator=(UIChar &&) = delete;
+		virtual ~UIChar(void) = default;
+		
+		std::string getName(void) const;
+		void setScore(int);
```

Comment:
Ok, in that case everything about `UIChar` can wait until I have the basic entities in place (before Sunday I think, maybe even this afternoon).